bitnami/jupyterhub/templates/hub/secret.yaml

```
@@ -0,0 +1,18 @@
+{{- if not .Values.hub.existingSecret }}
+apiVersion: v1
+kind: Secret
+metadata:
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: hub
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  name: {{ template "common.names.fullname" . }}-hub
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+type: Opaque
+data:
+  values.yaml: {{ include "common.tplvalues.render" ( dict "value" .Values.hub.configuration "context" $ ) | b64enc | quote }}
+  proxy-token: {{ include "jupyterhub.proxyToken" . | b64enc | quote }}
```

I don't see necessary a macro for this:
```suggestion
  proxy-token: {{ default (randAlphaNum 32) .Values.proxy.secretToken | b64enc | quote }}
```